lib/galaxy/config.py

```diff
@@ -276,6 +276,11 @@ def __init__( self, **kwargs ):
         # On can mildly speed up Galaxy startup time by disabling index of help,
         # not needed on production systems but useful if running many functional tests.
         self.index_tool_help = string_as_bool( kwargs.get( "index_tool_help", True ) )
+        self.tool_name_boost = kwargs.get( "tool_name_boost", None )
```

Comment:
If multiple components needed to access these values - then I would move the defaults into config and rely more heavily on the unfortunate dependency between the component and the ``config`` God-object. This isn't the case though - no need to prematurely create the dependencies in my mind.